azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/models/CallCompositeLocalOptions.java

```diff
@@ -33,14 +35,21 @@
 public final class CallCompositeLocalOptions {
     private final CallCompositeParticipantViewData participantViewData;
 
+    private final String callTitle;
+
+    private final String callSubTitle;
+
     /**
      * Create LocalSettings.
      *
      * @param participantViewData The {@link CallCompositeParticipantViewData};
      * @see CallCompositeParticipantViewData
      */
-    public CallCompositeLocalOptions(final CallCompositeParticipantViewData participantViewData) {
+    public CallCompositeLocalOptions(final CallCompositeParticipantViewData participantViewData,
```

Comment:
How about we will add a default constructor that does not take any parameters and add settert to all fields?

Reply:
@pavelprystinka  Fixed in d1ea113